ep/src/lib.rs

```diff
@@ -78,6 +85,15 @@ impl Ep {
   }
 }
 
+impl EpOpts {
+  fn new() -> EpOpts {
+    EpOpts {timeout: 1, unit: 1, affinity: 1, shm_mbytes: 1,
```

Comment:
Typical idiomatic style for these is 
```rust
EpOpts {
  stuff: 1
  other_stuff: 1
}
```

You can typically condense those values several per line like you have, so:
```rust
EpOpts {
  stuff: 1, other_stuff: 2
}
```

Any style is fine ,just say consistent.